.husky/reuse

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if command -v reuse &> /dev/null
+then
+    (cd databrowser && reuse lint)
```

Comment:
`reuse` must be invoked on the root of the project in order for it to find the `.git` folder. We must not change to the `databrowser` folder before running `reuse`.

The line should just be `reuse lint`

Reply:
Yes i noticed that too, interestingly the lint executes the exact same way, but I will change it anyways.